MasterWAMPServer.spec.js

```diff
@@ -1,46 +1,47 @@
-'use strict';
+/* eslint no-unused-expressions: 0 */
+// https://github.com/jonathanglasmeyer/graphql-sequelize/commit/00814cac3aa9fa6d20aed38df838dcbc4b4ab9b4
 
 const chai = require('chai');
 const sinon = require('sinon');
 const MasterWAMPServer = require('./MasterWAMPServer');
 const MasterWAMPRequestSchema = require('./schemas').MasterWAMPRequestSchema;
-const MasterWAMPResponseSchema = require('./schemas').MasterWAMPResponseSchema;
-const expect = chai.expect;
 
-describe('MasterWAMPServer', function () {
+const expect = chai.expect;
```

Comment:
but now i have to use chai anyway to use `dirtyChai`: `chai.use(dirtyChai);` so I will leave it as it is